components/logins/src/store.rs

```diff
@@ -126,11 +141,16 @@ impl PasswordStore {
         &self,
         storage_init: &Sync15StorageClientInit,
         root_sync_key: &KeyBundle,
+        local_encryption_key: &str,
     ) -> Result<telemetry::SyncTelemetryPing> {
-        let engine = LoginsSyncEngine::new(&self);
-
+        let mut engine = LoginsSyncEngine::new(&self);
         let mut disk_cached_state = engine.get_global_state()?;
         let mut mem_cached_state = self.mem_cached_state.take();
+        // TODO-sqlcipher: unwrap() is not right here, but the above comment makes me think we
+        // should not worry about this too much yet.
+        engine
+            .set_local_encryption_key(local_encryption_key)
```

Comment:
@mhammond Just out of curiosity, what prevents us from removing this `sync` function and relying on sync manager for logins. If it's not too much of a lift, it might be worth removing this function so this `set_local_encryption_key` function isn't necessary.